environment.yml

```diff
@@ -25,8 +25,10 @@ dependencies:
   - multidict
   - frozendict
   - nb_conda_kernels
+  - pip
   - pip:
     - "git+https://github.com/inducer/codepy"
-    - pyrevolve==1.0.2
+    - pyzfp>=0.2
```

Comment:
so compression becomes more and more a first class citizen in devito and I'm happy about that, we should probably just advertise it more